.github/workflows/build.yml

```
@@ -0,0 +1,89 @@
+        stack install
+        
+    - uses: actions/upload-artifact@v1
+      with:
+        name: wstunnel-windows-x64.exe
+        path: C:\Users\runneradmin\AppData\Roaming\local\bin\wstunnel.exe
+      
+      
+  build-macos:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)